apps/image-focal-point/src/index.js

```diff
@@ -52,18 +52,18 @@ export class App extends React.Component {
     const value = e.currentTarget.value;
     this.setState({ value });
     if (value) {
-      this.props.sdk.field.setValue(value);
+      this.props.sdk.field.setValue(value, this.findProperLocale());
```

Comment:
Shouldn't the app always update itself? In which scenario do we need the locale here?